scripts/cli-test.sh

```diff
@@ -45,7 +45,7 @@ $FM_MINT_CLIENT fetch
 
 # peg out
 PEG_OUT_ADDR="$($FM_BTC_CLIENT getnewaddress)"
-$FM_MINT_CLIENT peg-out --address $PEG_OUT_ADDR --satoshis 500
+$FM_MINT_CLIENT peg-out --address $PEG_OUT_ADDR --amount 500
```

Comment:
Are accepting `Amount` here? We should use units, just for clarity, so `500sat`, since `--amount` doesn't tell use the unit anymore. But anyway - not exactly

Reply:
Yup, that makes sense.